Sources/RDeviceIdentifier/RDeviceIdentifier.swift

```diff
@@ -0,0 +1,283 @@
+    /// benefits:
+    ///
+    /// Feature                                                       | `-[UIDevice identifierForVendor]` | `-[RDeviceIdentifier uniqueDeviceIdentifier]`
+    /// ------------------------------------------------------------- | --------------------------------- | ------------------------------------------------
+    /// Universally unique                                            | YES                               | YES
+    /// Restored from device backups, but only on the original device | YES                               | YES
+    /// Survives OS update                                            | YES                               | YES
+    /// Survives application update                                   | YES                               | YES
+    /// Survives application uninstall                                | YES                               | YES
+    /// Survives all applications being uninstalled ¹                 | NO                                | YES
+    /// Works without 3-component bundle ID ²                         | NO                                | YES
+    ///
+    /// 1. If a Rakuten application is reinstalled after all have been uninstalled,
+    ///    iOS resets the value to be returned by `-[UIDevice identifierForVendor]`.
+    /// 2. `-[UIDevice identifierForVendor]` uses the application's bundle identifier
+    ///    to determine whether two applications come from the same publisher, but iOS6 and iOS7
+    ///    do things differently and the only way to get a consistent behavior across OS versions
+    ///    with that method is to use a 3-component bundle ID, e.g. `com.rakuten.ichiba`.
+    ///
+    /// - Warning: Applications built with different application identifier prefixes/bundle seed identifiers, i.e.
+    ///          different provisioning profiles, will not produce the same device identifier.
+    ///
+    /// - Returns: A string uniquely identifying the device the application is currently running on.
+    ///         If the keychain is not available (i.e. the device is locked) and no value has been
+    ///         retrieved yet, `nil` is returned and the developer should try again when the device
+    ///         is unlocked.
+    ///
+    /// - Precondition: NSObjectInaccessibleException if the application's entitlements do not include the
+    ///            access group required to access the device identifier. See @ref device-information-keychain-setup "Setting up the keychain"
+    ///            for more information.
+    /// - Precondition: NSInternalInconsistencyException if the application is misconfigured and the first
+    ///            access group does not match the application's bundle identifier. See @ref device-information-keychain-setup "Setting up the keychain"
+    ///            for more information.
+    @objc public class var uniqueDeviceIdentifier: String? {
```

Comment:
Is there a reason to declare it as `class` instead of `static`?
Do you use `class` in order to be able to override`uniqueDeviceIdentifier`?